src/components/modules/MediaForm.svelte

```diff
@@ -68,8 +70,10 @@
  .module_topbar .module_title {
      display: flex;
      font-size: 16px;
-     transform: translateX(30px) translateY(25px) rotate(90deg);
+     transform: translateX(35px) translateY(30px) rotate(90deg);
```

Comment:
Xaiki: i'm affraid these px translations will break on different screensize/resolutions/dpi

Reply:
I've tested it in different sceensizes and it seems to be working. Also it's probable the app will only be used in Desktop devices. I'm not sure if we could use percentages instead of pixels as an alternative.